src/YieldDistributor.sol

```diff
@@ -66,8 +69,15 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
         address[] memory _projects
     ) public initializer {
         __Ownable_init(msg.sender);
-
+        if (
```

Comment:
I think this is more readable personally
```
if (
    _bread == address(0) ||
    _butteredBread == address(0) ||
    _precision == 0 ||
    _minRequiredVotingPower == 0 ||
    _maxPoints == 0 ||
    _cycleLength == 0 ||
    _yieldFixedSplitDivisor == 0 ||
    _lastClaimedBlockNumber == 0 ||
    _projects.length == 0
) {
    revert MustBeGreaterThanZero();
}
```

Reply:
the formatter will override this if I attempt to input that ...

Reply:
formatter overrides this.. do you know if there's a comment I can add for it to ignore this?

Reply:
Aside from shorting `line_length`, which i wouldn't recommend, not really. all good